proposals/0029-cooperative-vector.md

```diff
@@ -89,6 +89,8 @@ ByteAddressBuffer biasVector1;
 
 void ps_main(args) // args: texture, normal, position
```

Comment:
Also on line 80, the spec states that this is HLSL-like pseudocode. Now that we're aiming to express what it should really be using the proposed API, this could be tweaked (Maybe "The below shader is the proposed HLSL needed to express the idea ...". 
(nit: pseudocode spelt wrong)

Reply:
Updated